packages/ripple-binary-codec/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ripple-binary-codec",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Comment:
Why do we need to increment the version of `ripple-binary-codec` package?

Reply:
For `definitions.json` changes.